endpoints.json

```diff
@@ -1,27 +1,98 @@
-{
+{ "summary of all endpoints available on this API" : {
   "GET /api": {
     "description": "serves up a json representation of all the available endpoints of the api"
   },
   "GET /api/topics": {
-    "description": "serves an array of all topics",
-    "queries": [],
+    "description": "responds with an array of all topics",
     "exampleResponse": {
       "topics": [{ "slug": "football", "description": "Footie!" }]
     }
   },
   "GET /api/articles": {
-    "description": "serves an array of all topics",
-    "queries": ["author", "topic", "sort_by", "order"],
+    "description": "responds with an array of all topics",
+    "queries": ["topic", "sort_by", "order"],
     "exampleResponse": {
       "articles": [
         {
-          "title": "Seafood substitutions are increasing",
+          "title": "Mob attacks McDonald's restaurant for not serving breakfast after 10am.",
           "topic": "cooking",
-          "author": "weegembump",
+          "author": "axXXyJuniOr$#$$£",
           "body": "Text from the article..",
           "created_at": 1527695953341
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id" : {
+    "description": "responds with the article object specified in the request URL",
+    "exampleResponse": {
+        "article": {
+          "author": "icellusedkars",
+          "title": "Does Mitch predate civilisation?",
+          "article_id": 8,
+          "body": "Latest projections place the total rainfall at ...",
+          "topic": "mitch",
+          "created_at": "2020-04-17T01:08:00.000Z",
+          "votes": 0,
+          "comment_count": 0
+        }
+      }
+  },
+  "GET /api/articles/:article_id/comments" : {
+    "description": "responds with an array of comments for the article specified in the request URL",
+    "exampleResponse": {
+      "comments": [
+        {
+          "comment_id": 15,
+          "votes": 1,
+          "created_at": "2020-11-24T00:08:00.000Z",
+          "author": "butter_bridge",
+          "body": "I am 100% sure that we're not completely sure."
+        }
+      ]
+    }
+  },
+  "POST /api/articles/:article_id/comments" : {
+    "description": "enables client to add a new comment to the database and returns the new entry",
```

Comment:
You'll thank me when we get to front end, but i'd add an exampleRequest as well. 

Explain what the post/patch object needs to look like in order for it to be accepted